scylla/src/frame/types.rs

```diff
@@ -33,6 +33,12 @@ pub enum SerialConsistency {
     LocalSerial = 0x0009,
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub enum ConsistencyResult {
```

Comment:
The `Result` suffix in the type's name is confusing as it may suggest that it is related in some way to the `std::result::Result`.

I'm not sure if I have a better suggestion for a name... `GeneralConsistency` maybe? `QueryConsistency`? Might also be good to add a comment explaining the need for this struct...

Reply:
Yeah, I'm also not a fan, I just didn't figure out a better name - I wanted to somehow express the fact that this type is only needed for parsing something that the server sent to us. This type is ultimately only used to get printed in certain error messages, so maybe I'll call it `LegacyConsistency`, to express the fact that it used to be a single type?

Reply:
OK, let's proceed with `LegacyConsistency`.